app/views/chefs/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Chefs#new</h1>
+<p>Find me in app/views/chefs/new.html.erb</p>
+
+<%= simple_form_for [ chef ], url: new_chef_path, method: :post do |form| %>
```

Comment:
I think your line should be changed to the following:
<%= simple_form_for @chef do |form| %>

Reply:
Thx, fixed it!